### ISSUE TITLE
UI fails to render analytics for https

### ISSUE THREAD

**Issue by fogfish** [zalando/typhoon#110]
```
request() ->
   do([m_http ||
      _ /= new("https://127.0.0.1:8080/health/peer?q=1"),
      _ /= x('GET'),
      _ /= h("Connection", "keep-alive"),
      _ /= r(),
      return(_)
   ]).
```

**Comment by fogfish** [comment]
Only latest master branch is impacted.

**Comment by fogfish** [comment]
fixed at release 1.0.0
